server/src/models/blogs.js

```diff
@@ -6,27 +6,23 @@ const BlogsSchema = new Schema({
   userId: {
     type: Schema.Types.ObjectId,
     ref: 'User',
+    required: true,
   },
   title: {
     type: String,
     required: true,
   },
-  content: [
-    {
-      _id: Schema.Types.ObjectId,
-      type: String,
-      fragment: {
-        type: String,
-        required: true,
-      },
-    },
-  ],
+  content: {
+    type: String,
+    required: true,
+  },
   likes: {
     type: [Schema.Types.ObjectId],
     ref: 'User',
   },
   private: {
     type: Boolean,
+    default: false,
   },
 }, {
   timestamps: true,
```

Comment:
Please wait for the updated schema P.R. to be merged. That is going to bring in a lot of changes in the given schema. :smile: